addons/web/static/tests/fields/relational_fields_mobile_tests.js

```diff
@@ -116,8 +118,236 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.module('FieldMany2One');
+
+    QUnit.test("many2one in a mobile environment", async function (assert) {
```

Comment:
we should move this test in enterprise too.

Reply:
same as above